frontend/src/app/modules/metric-finder/components/filmstrip-component/filmstrip.component.scss

```diff
@@ -0,0 +1,12 @@
+.filmstrip-container {
+  display: flex;
+  background: black;
+  overflow-y: auto;
```

Comment:
why overflow-y? The height should be dynamic, since you didn't define it.

Overflow-x would be important to have a scroll bar (but that might be flexbox's default if it works already).

Reply:
just removing overflow-y also removes the scrollbar, replaced it with overflow-x